includes/class-wc-payments-features.php

```diff
@@ -233,7 +234,10 @@ public static function is_progressive_onboarding_enabled(): bool {
 	 * @return  bool
 	 */
 	public static function is_fraud_protection_settings_enabled(): bool {
-		return '1' === get_option( 'wcpay_fraud_protection_settings_active', '0' );
+		$account    = WC_Payments::get_database_cache()->get( WCPay\Database_Cache::ACCOUNT_KEY );
+		$is_enabled = is_array( $account ) && ( $account['is_fraud_protection_settings_enabled'] ?? false );
```

Comment:
Shouldn't we set the default as enabled? If `is_fraud_protection_settings_enabled` key doesn't exist on the account object (I have doubts that it won't), but in the meantime the server part of it is being coded, the feature should be enabled.

Reply:
@tpaksu I want us to enable this from the server, so I made the default disabled. The fallback that we can use in testing is the option: `_wcpay_fraud_protection_settings_enabled`.

Reply:
On the server we're going to return `true` if the account meta that disables the rule engine doesn't exist then. But I don't think that's what we want, modifying this feature flag will hide the whole UI from the customer. Is that what we want?

Reply:
Oh yes, initially, we wanted to hide it. I'm doing it this way so we can control the rollout without relying on a client release. We can turn it on from the server side and control who sees it. 

This relates to paJDYF-7ra-p2 but since the DB-based flag will be a bit much to implement, I'm opting for a code-level flag.